src/app/datagrid/hide-show-columns/hide-show.html

```diff
@@ -77,6 +77,11 @@ <h2>Datagrid: Hide-show columns demo</h2>
     </clr-dg-row>
 
     <clr-dg-footer>
+        <clr-dg-column-toggle>
+            <clr-dg-column-toggle-title>Show Columns!!!</clr-dg-column-toggle-title>
+            <clr-dg-column-toggle-button [type]="'selectAll'">Select All!!!</clr-dg-column-toggle-button>
+            <clr-dg-column-toggle-button [type]="'ok'">OK!!!</clr-dg-column-toggle-button>
```

Comment:
You had an opportunity to put German in our sample app! Too bad. I like 2-lines buttons. 😛 

(Everyone knows an "OK" button becomes "Ich stimme dieser Entscheidung zu" in German. :trollface: )

Reply:
I added German, and an icon for fun.